doc/concepts/probe_sample.rst

```diff
@@ -3,8 +3,55 @@
 Cable cell probing and sampling
 ===============================
 
-.. TODO::
-    WIP!
+Definitions
+***********
+
+.. glossary::
+
+    probe
+        A measurement that can be performed on a cell. Each cell kind will have its own sorts of probe;
+        Cable cells (:py:attr:`arbor.cable_probe`) allow the monitoring of membrane voltage, total membrane
+        current, mechanism state, and a number of other quantities, measured either over the whole cell,
+        or at specific sites (see :ref:`pycablecell-probesample`).
+        
+    probeset
+        A set of probes. Probes are placed on locsets, and therefore may describe more than one probe.
+
+    probeset address
+        Probesets are located at a probeset address, and the collection of probeset addresses for a given cell is
+        provided by the :py:class:`recipe` object. One address may correspond to more than one probe:
+        as an example, a request for membrane voltage on a cable cell at sites specified by a location
+        expression will generate one probe for each site in that location expression.
+
+        See :ref:`pycablecell-probesample` for a list of objects that return a probeset address.
+
+    probeset id
+        A designator a probeset as specified by a recipe. The *probeset id* is a
+        :py:class:`cell_member` referring to a specific cell by gid, and the index into the list of
+        probeset addresses returned by the recipe for that gid.
+
+    metadata
+        Each probe has associated metadata describing, for example, the location on a cell where the
+        measurement is being taken, or other such identifying information. Metadata for the probes
+        associated with a :term:`probeset id` can be retrieved from the simulation object, and is also provided
+        along with any recorded samples.
+
+    sampler
+        A sampler is something that receives probeset data. It amounts to setting a particular :term:`probeset` to a
```

Comment:
Here `probeset` is used once w/ a 'term` prefix and once w/o. Accident?

Reply:
Yes.